_haikus/The_Hunt.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: The Hunt
+author: Hanike
+---
+
+Hunt in the darkness<br>
+A cold presence watches you fall<br>
```

Comment:
This line is just 1 syllable too long, you can probably remove the word `fall` here

Reply:
Oh, I used Heroku's haiku validator to check it and it was counting it right.. Gonna remove that first "A" then. I wasn't sure what the Travis-CI was saying it was the error coz it showed like 9 errors that didn't make sense for me. Thanks! =)